src/styles/main.scss

```diff
@@ -1,26 +1,26 @@
 /* ------ VI colors, done by Philip ------------- */
```

Comment:
Maybe remove this 'done by Philip'?

Reply:
we can add Zuri here and leave Philip here, as he was the one doing it from the start

Reply:
any solution is fine for me